org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/extensions/contentmodel/XMLSchemaHoverDocumentationTypeTest.java

```diff
@@ -301,6 +325,28 @@ private void assertElementMultipleBothHover(String expected, SchemaDocumentation
 		assertHover(xml, expected, docSource, markdownSupported);
 	}
 
+	private void assertElementHoverNoAnnotation(SchemaDocumentationType docSource, boolean markdownSupported) throws BadLocationException {
+		String xml =
+				"<root\n" + //
+				"	xmlns=\"http://docAppinfo\"\n" + //
+				"	xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"\n" + //
+				"	xsi:schemaLocation=\"http://docAppinfo xsd/" + schemaName + "\"\n" + //
+				"	<e|lementNoAnnotation></elementNoAnnotation>\n" + //
+				"</root>\n";
+		assertHover(xml, null, docSource, markdownSupported);
+	}
+
+	private void assertElementHoverWhitespaceAnnotation(SchemaDocumentationType docSource, boolean markdownSupported) throws BadLocationException {
+		String xml =
+				"<root\n" + //
+				"	xmlns=\"http://docAppinfo\"\n" + //
+				"	xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"\n" + //
+				"	xsi:schemaLocation=\"http://docAppinfo xsd/" + schemaName + "\"\n" + //
+				"	<e|lementWhitespaceAnnotation></elementWhitespaceAnnotation>\n" + //
+				"</root>\n";
+		assertHover(xml, null, docSource, markdownSupported);
+	}
+
 	private void assertHover(String xml, String expected, SchemaDocumentationType docSource, boolean markdownSupported) throws BadLocationException {
 		String currSource = markdownSupported ? source : plainTextSource;
```

Comment:
Could probably move this inside the if statement since the else block is not needed anymore